pycheribuild/config/compilation_targets.py

```diff
@@ -256,6 +256,18 @@ def triple_for_target(cls, target: "CrossCompileTarget", config: "CheriConfig",
         # TODO: do we need any special cases here?
         return target.cpu_architecture.value + common_suffix
 
+    @property
+    def freebsd_target_arch(self):
+        mapping = {
+            CPUArchitecture.AARCH64: "aarch64",
+            CPUArchitecture.ARM32: "armv7",
```

Comment:
Wasn't sure whether to add this or not, but I thought might as well give it a sensible value.

Reply:
LGTM